build.ps1

```diff
@@ -9,7 +14,7 @@ if ($IsMacOS) { $runtime = 'osx-x64' }
 elseif ($IsLinux) { $runtime = 'linux-x64' }
 else { $runtime = 'win-x64' }
 
-dotnet publish src/Azure.Functions.Cli/Azure.Functions.Cli.csproj --runtime $runtime --output $outDir
+dotnet publish src/Azure.Functions.Cli/Azure.Functions.Cli.csproj --runtime $runtime --output $outDir -c $config
```

Comment:
Are the changes to this file related? They don't seem to be. But if @ahmelsayed is okay with these, go for it.

Reply:
it's a dev build script, so it's fine :)

Reply:
Hello , i am still facing this issue. not able to see Traces from azure function to application insights

Reply:
> Hello , i am still facing this issue. not able to see Traces from azure function to application insights

If you're not seeing them in appinsights it's likely because you don't [have monitoring configured correctly](https://docs.microsoft.com/en-us/azure/azure-functions/functions-monitoring?tabs=cmd). This PR did not affect logging to AI; just to console.